specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-11-15/timeseriesinsights.json

```diff
@@ -1353,6 +1393,53 @@
         }
       }
     },
+    "EnvironmentStatus": {
+      "readOnly": true,
+      "type": "object",
+      "description": "An object that represents the status of an environment and its associated operations.",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "ingress": {
+          "$ref": "#/definitions/IngressEnvironmentStatus"
+        }
+      }
+    },
+    "IngressEnvironmentStatus": {
+      "readOnly": true,
+      "type": "object",
+      "description": "An object that represents the status of ingress on an environment.",
+      "properties": {
+        "state": {
+          "type": "string",
+          "description": "This string represents the state of ingress operations on an environment. It can be \"Disabled\", \"Ready\", \"Running\", \"Paused\" or \"Unknown\"",
+          "enum": [ "Disabled", "Ready", "Running", "Paused", "Unknown" ],
```

Comment:
It's a nit, but for other enums we have listed them with single value per line. I think I prefer that, probably just because of the resemblance to style in C-languages.